com.wso2telco.dep.common.registry.resources/mediationConfig.xml

```diff
@@ -27,4 +27,6 @@
 	<appendTokenHeader>false</appendTokenHeader>
 	<!-- This must be set if above is true -->
 	<tokenHeaderName></tokenHeaderName>
+	<!-- This is the key of user mask decription key -->
+	<userMaskingSecretKey>changethis</userMaskingSecretKey>
```

Comment:
Can remove this property from this mediationConfig.xml file since this property has been moved to the property file